docs/Component/PropTypes.md

```diff
@@ -1,3 +1,28 @@
 # PropTypes
 
-This documentation isn't finished yet! In fact, this project is supposed to be very hush hush. Come back after we officially release!
+
+The shasta PropTypes inherit from [React PropTypes](https://facebook.github.io/react/docs/reusable-components.html#prop-validation) as well as [react-immutable-proptypes](https://github.com/HurricaneJames/react-immutable-proptypes) to add support for immutable validators. Please view the [react-immutable-proptypes documentation](https://github.com/HurricaneJames/react-immutable-proptypes/blob/master/README.md) for more information.
```

Comment:
All of the PropTypes should be documented here. Linking to external docs is a bad idea (learned this with gulp) as it gets extremely confusing.